src/views/common/Animation/AnimationStateReducer.tsx

```diff
@@ -63,6 +65,10 @@ export type AnimationStateSetReplayRateAction = {
     newRate: number
 }
 
+export type AnimationStateReverseReplayRateAction = {
+    type: 'REVERSE_REPLAY_RATE'
+}
+
```

Comment:
This had been implemented by doing a `SET_REPLAY_RATE` with the value equal to -1 times the previous rate, but I thought it would be neater to have the reducer just take care of that.